test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3029,7 +3030,41 @@ static public void testUnresolvedPutFieldUse() throws Throwable {
 			}
 		}
 	}
-	
+
+	@Test(priority = 1)
+	static public void testIdentityObjectOnValueType() throws Throwable {
+		String fields[] = {"longField:J"};
+		Class valueClass = ValueTypeGenerator.generateValueClass("testIdentityObjectOnValueType", fields);
+		assertFalse(Arrays.asList(valueClass.getInterfaces()).contains(IdentityInterface.class));
+	}
+
+	@Test(priority = 1)
+	static public void testIdentityObjectOnAbstract() throws Throwable {
+		assertFalse(Arrays.asList(AbstractClass.class.getInterfaces()).contains(IdentityInterface.class));
+	}
+	private static abstract class AbstractClass {
```

Comment:
@tajila I created an abstract class here to check if it contains the IdentityInterface. Is there another way to test the abstract class, or is this fine? Would I want to populate the class with methods and fields, or simply leave it blank?

Reply:
I have also made them priority = 1 since they don't rely on any other methods. Lmk if that should change.

Reply:
@tajila